### PR TITLE
fix include guard for test

### PR DESCRIPTION
**Pull request by altairpearl**
Refering to the issue #2623. Have modified the test group include guards . Please review .

### CODE REVIEW

File: tests/unittests/tests-ipv6_netif/tests-ipv6_netif.h [hunk 1]
```diff
@@ -15,8 +15,13 @@
  *
  * @author      Martine Lenders <mlenders@inf.fu-berlin.de>
  */
+<<<<<<< Updated upstream:tests/unittests/tests-ipv6_netif/tests-ipv6_netif.h
 #ifndef TESTS_IPV6_NETIF_H_
 #define TESTS_IPV6_NETIF_H_
+=======
+#ifndef TESTS_NOMAC_H_
+#define TESTS_NOMAC_H_
+>>>>>>> Stashed changes:tests/unittests/tests-nomac/tests-nomac.h
```
Comment (gebart) [inline tests/unittests/tests-ipv6_netif/tests-ipv6_netif.h]: You had a merge conflict which you did not resolve, fix this.

File: tests/unittests/tests-ipv6_netif/tests-ipv6_netif.h [hunk 2]
```diff
@@ -33,5 +38,9 @@ void tests_ipv6_netif(void);
 }
 #endif
 
+<<<<<<< Updated upstream:tests/unittests/tests-ipv6_netif/tests-ipv6_netif.h
 #endif /* TESTS_IPV6_NETIF_H_ */
+=======
+#endif /* TESTS_NETDEV_DUMMY_H_ */
+>>>>>>> Stashed changes:tests/unittests/tests-nomac/tests-nomac.h
```
Comment (gebart) [inline tests/unittests/tests-ipv6_netif/tests-ipv6_netif.h]: merge conflict

**Comment by gebart** [comment]
ACK, after you remove the changes to  `tests/unittests/tests-ipv6_netif/tests-ipv6_netif.h`.

**Comment by gebart** [comment]
You need to change the commit message to include a module name, e.g. "tests: Updated include guards to remove leading underscores"

**Comment by altairpearl** [comment]
ok got it! Will work on it now .

**Comment by gebart** [comment]
Why did you close this issue?

You can add more commits, rework and refactor your current PR by pushing to your existing PR branch.

**Comment by altairpearl** [comment]
Oh ok. I get it. Sorry for the mistake.

**Pull request closed without merging**